src/cli/src/lib.rs

```diff
@@ -252,13 +253,23 @@ impl FluvioCmd {
 struct VersionOpt {}
 
 impl VersionOpt {
-    pub fn process(self) -> Result<()> {
-        println!("Fluvio version : {}", crate::VERSION);
-        println!("Git Commit     : {}", env!("GIT_HASH"));
+    pub async fn process(self, target: ClusterTarget) -> Result<()> {
+        println!("Fluvio CLI      : {}", crate::VERSION.trim());
+
+        // Attempt to connect to a Fluvio cluster to get platform version
+        // Even if we fail to connect, we should not fail the other printouts
+        if let Ok(fluvio_config) = target.load() {
+            if let Ok(fluvio) = Fluvio::connect_with_config(&fluvio_config).await {
+                let version = fluvio.platform_version();
+                println!("Fluvio Platform : {}", version);
+            }
```

Comment:
in case of error,  we should just print out error instead of swallowing

Reply:
Well suppose that I am a new Fluvio user and I have never connected the CLI to a cluster yet. I try to run `fluvio version`. I would expect it not to crash or give me an error. All this is doing is optionally printing the cluster SC version info only when we can successfully connect to a cluster. Otherwise it prints just local version info.

Reply:
In that case, just say `Problem connecting to Fluvio`.    For kubectl, this happens when it can't connects
```
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.0", GitCommit:"e19964183377d0ec2052d1f1fa930c4d7575bd50", GitTreeState:"clean", BuildDate:"2020-08-26T14:30:33Z", GoVersion:"go1.15", Compiler:"gc", Platform:"linux/amd64"}
The connection to the server 192.168.49.2:8443 was refused - did you specify the right host or port?
``